app/views/providers/visas/edit.html.erb

```diff
@@ -0,0 +1,44 @@
+<%= content_for :page_title, "#{@form_object.errors.present? ? 'Error: ' : ''}Visa sponsorship" %>
+<% content_for :before_content do %>
+  <%= govuk_back_link_to(details_provider_recruitment_cycle_path(@provider.provider_code, @provider.recruitment_cycle_year)) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with(
+      model: @form_object,
+      url: provider_recruitment_cycle_visas_path(@provider.provider_code, @provider.recruitment_cycle_year),
+      method: :post,
+      scope: "",
+      builder: GOVUKDesignSystemFormBuilder::FormBuilder,
+    ) do |form| %>
+
+      <%= form.govuk_error_summary %>
+
+      <h1 class="govuk-heading-l">
+        <span class="govuk-caption-l"><%= @provider.provider_name %></span>
+        Visa sponsorship
+      </h1>
+
+      <%= form.govuk_radio_buttons_fieldset(
+        :can_sponsor_student_visa,
+        legend: { text: "Can you sponsor Student visas?" },
+        hint: { text: "Applies to fee-paying courses" },
+      ) do %>
+        <%= form.govuk_radio_button :can_sponsor_student_visa, true, label: { text: "Yes" }, link_errors: true %>
+        <%= form.govuk_radio_button :can_sponsor_student_visa, false, label: { text: "No" } %>
+      <% end %>
+
+      <%= form.govuk_radio_buttons_fieldset(
+        :can_sponsor_skilled_worker_visa,
+        legend: { text: "Can you sponsor Skilled Worker visas?" },
+        hint: { text: "Applies to salaried courses" },
+      ) do %>
+        <%= form.govuk_radio_button :can_sponsor_skilled_worker_visa, true, label: { text: "Yes" }, link_errors: true %>
+        <%= form.govuk_radio_button :can_sponsor_skilled_worker_visa, false, label: { text: "No" } %>
+      <% end %>
+
+      <%= form.govuk_submit "Save" %>
```

Comment:
![image](https://user-images.githubusercontent.com/470137/123062568-8d450300-d404-11eb-9dec-1771c24c5f9d.png)

I think the wording and CTA should be inline with the other pages unless it don't get published

Reply:
Makes sense - the changes take effect immediately so I guess they are saved and published